internal/controller/rbac/definition/roles.go

```diff
@@ -48,7 +48,8 @@ const (
 
 	valTrue = "true"
 
-	suffixStatus = "/status"
+	suffixStatus     = "/status"
+	suffixFinalizers = "/finalizers"
```

Comment:
Does this sub resource exist? Is it a special OpenShift thing?

I thought it was something being proposed upstream, but not yet implemented. e.g. https://github.com/kubernetes/enhancements/issues/3285

Reply:
@negz thank you for involving
I'm not sure if exist subresource management in OpenShift, but It seems rights on finalizer CR's needed. 
I can check it

P.S. By default, OwnerReferencesPermissionEnforcement is enabled https://docs.openshift.com/container-platform/4.10/architecture/admission-plug-ins.html#admission-plug-ins-default_admission-plug-ins

Reply:
Right, I'm familiar with finalizers, what I've never seen before is the `someresource/finalizers` as a valid RBAC resource. I thought this had to be for [subresources](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#subresources) like `/status`.

I see from other PRs that this is a real thing, so I'm just curious whether it affects upstream Kubernetes or just OpenShift.